### DISCUSSION TITLE
How to disable https inside docker

### DISCUSSION THREAD

**Discussion by kri164** [stalwartlabs/jmap-server#33] [Q&A] [answered]
Hi,

I started docker container behind proxy. Proxy do https termination. When trying to connect from web browser, I get error

Client sent an HTTP request to an HTTPS server.

How to I disable https inside docker container?

Command used to start docker
```
podman run -d -ti -p 11200:11200 \
           -v /run/containers/jmap:/usr/local/stalwart-jmap \
           --name stalwart-jmap stalwartlabs/jmap-server:latest \
           --lmtp-bind-addr=0.0.0.0 \
           --jmap-url=https://jmap.example.org \
           --db-path=/usr/local/stalwart-jmap/data \
           --encryption-key="[redacted]
```

Caddy proxy has this configuration
```
https://jmap.example.org:443 {
        reverse_proxy localhost:8080
}
```

Thank you

**Comment by mdecimus** [comment, marked as answer]
Hi,

HTTPS is disabled automatically if you do not provide a TLS certificate with the ``jmap-cert-path`` and ``jmap-key-path`` arguments.

**Reply by kri164** [reply]
OK.

It was my fault to not map all needed docker ports correctly.
Should be -p 443:8080 -p 11200:11200

Thank you for reply
